frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -594,6 +595,18 @@ func toDispatchState(ctx context.Context, dt []byte, opt ConvertOpt) (*dispatchS
 	buildContext := &mutableOutput{}
 	ctxPaths := map[string]struct{}{}
 
+	var dockerIgnoreMatcher *patternmatcher.PatternMatcher
+	dockerIgnorePatterns, err := opt.Client.DockerIgnorePatterns(ctx)
```

Comment:
Move this inside the `if opt.Client.CopyIgnoredCheckEnabled`

Reply:
Done!